### ISSUE TITLE
Comparison to iamdustan/smoothscroll

### ISSUE THREAD

**Issue by aldendaniels** [wessberg/scroll-behavior-polyfill#11]
Hi, thanks for creating this library. I'm evaluating this pollyfill vs [iamdustan/smoothscroll](https://github.com/iamdustan/smoothscroll/blob/master/src/smoothscroll.js). Dustin's library is older has much more usage. What inspired you to write `scroll-behavior-polyfill`? How does it compare to dustin's library? 

FWIW I appreciate that `scroll-behavior-polyfill` is written in typescript. A comparison vs alternatives would be a useful addition to the README. 

Thanks!

**Comment by wessberg** [comment]
Hey there! I decided to implement this polyfill since there were no alternatives that properly supported Shadow DOM at the time and generally none of the existing polyfills I evaluated were sufficiently spec-compliant. For example, the behavior of `scroll-behavior: auto` deviated in polyfill implementations from browser implementations, and not all prototype methods and accessors were being correctly polyfilled. Generally I strive for maximum specification compliance and seamless behavior in my polyfills and I hope you will find that to be the case during your evaluation.
